terraform/ecs.tf

```diff
@@ -118,39 +118,95 @@ resource "aws_autoscaling_group" "cluster" {
 
   tag {
     key                 = "Name"
-    value               = "${var.cluster_name}-ecs"
+    value               = "${var.cluster_name}-ecs-x86-64"
     propagate_at_launch = true
   }
 
   tag {
     key                 = "Cluster"
-    value               = var.cluster_name
+    value               = "${var.cluster_name}_X86_64"
     propagate_at_launch = true
   }
 }
 
-resource "aws_autoscaling_lifecycle_hook" "cluster" {
-  count = var.ecs_capacity_provider_enabled ? 0 : 1
-  name                   = "${var.cluster_name}-hook"
-  autoscaling_group_name = aws_autoscaling_group.cluster.name
+resource "aws_autoscaling_group" "cluster_arm64" {
```

Comment:
we'll need to merge this back into 1 resource with a for_each on the resource. There should be a variable "architectures" that defaults to ["x86_64"], but that can then be changed in the module to ["x86_64", "arm64"]